.snyk

```diff
@@ -5,5 +5,5 @@ ignore:
   SNYK-JAVA-IONETTY-1042268:
     - '*':
         reason: no available replacement
-        expires: 2021-12-31T00:00:00.000Z
+        expires: 2022-01-31T00:00:00.000Z
```

Comment:
what does this mean ?

Reply:
As currently there is no fixed version available for this (https://security.snyk.io/vuln/SNYK-JAVA-IONETTY-1042268), I am extending the expiry date for one month, till then it will not be reported as snyk failures. 

Do you see anything wrong or missing something? It's YYYY-MM-DD, right?